cardinal/server/server_test.go

```diff
@@ -381,6 +394,79 @@ func TestHandleSwaggerServer(t *testing.T) {
 	assert.NilError(t, err)
 	assert.Equal(t, resp6.StatusCode, 404)
 
+	//test query/game/cql
+	jsonQuery := struct{ CQL string }{"blah"}
+	jsonQueryBytes, err := json.Marshal(jsonQuery)
+	assert.NilError(t, err)
+	resp7, err := http.Post(txh.makeURL("query/game/cql"), "application/json", bytes.NewBuffer(jsonQueryBytes))
+	assert.NilError(t, err)
+	assert.Equal(t, resp7.StatusCode, 422)
+
```

Comment:
would it be possible to use a table driven test here? might be a bit nicer to deal w/

Reply:
hm my honest thoughts on this is that it's just tests. It doesn't matter too much as long as the tests are readable. copy + pasta is just more efficient here imo. 

But I don't really care, I can refactor this into a table.